packages/earth-map/src/components/last-viewed-place/last-viewed-place.tsx

```diff
@@ -18,15 +18,21 @@
 */
 
 import ListItem from 'components/list-item';
-import { IPlace } from 'modules/places/model';
 import React from 'react';
 
 interface IProps {
-  place: IPlace;
+  place: {
+    id: string;
+    slug?: string;
+    name?: string;
+    organization?: string;
+    mainType?: string;
```

Comment:
can we define mainType type as 'place' | 'collection' ? it's vague what this can be
same for subtype